elasticapm/instrumentation/packages/grpc.py

```diff
@@ -74,3 +76,30 @@ def call(self, module, method, wrapped, instance, args, kwargs):
         else:
             kwargs["interceptors"] = interceptors
         return wrapped(*args, **kwargs)
+
+
+class GRPCAsyncServerInstrumentation(AbstractInstrumentedModule):
+    name = "grpc_async_server_instrumentation"
+    creates_transactions = True
+    instrument_list = [("grpc", "aio.server")]
+
+    def get_instrument_list(self):
+        import grpc
+
+        # Check against the oldest version that I believe has the expected API
+        if parse_version(grpc.__version__) >= parse_version("1.33.1") and hasattr(grpc, "aio"):
```

Comment:
This is how I'm checking if we should try to instrument the async version, but I'm not sure what is the recommended way for doing this kind of thing here. I've thought of leaving just the check for the `aio` module's existence (and perhaps add a check for a `server` attribute inside `grpc.aio`), but since I wrote the version check first, I just left it there too.

Reply:
Actually, the whole check here is somewhat unnecessary. While instrumenting, we already check if the module/function we want to wrap is available or not. If we cannot import it, we simply skip it :)